api/v1_projects.inc

```diff
@@ -576,3 +591,22 @@ function api_v1_projects_holdstates($method, $data, $query_params)
 {
     return Project::get_holdable_states();
 }
+
+//---------------------------------------------------------------------------
+// Utility functions
+
+// Check to see if the user requested an enabled filter. This returns one of
+// three possible states:
+//   null - no enabled flag was set
+//   true - enabled flag was set to a non-"false" value
+//   false - enabled flag was set to "false"
+function _get_enabled_filter($query_params)
+{
+    if (!isset($query_params["enabled"])) {
+        return null;
+    } elseif ($query_params["enabled"] == "false") {
```

Comment:
should we make it case insensitive? Stack overflow also mentions filter_var https://stackoverflow.com/questions/4775294/parsing-a-string-into-a-boolean-value-in-php

Reply:
Yes, making it case-insensitive makes sense here.

Oh nice, I didn't know about `filer_var()`. Let me change things just a smidge.